CHANGELOG.rst

```diff
@@ -5,6 +5,26 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
+2.7.0 (2021-05-21)
+------------------
+
+**Added**
+
+* Make ``life.qbic.datamodel.dtos.business.ProductId`` comparable (`#211 <https://github.com/qbicsoftware/data-model-lib/pull/211>`_)
+
+* Add file extensions to names in Json examples (`#204 <https://github.com/qbicsoftware/data-model-lib/pull/204>`_)
+
+**Fixed**
+
+* Remove @CompileStatic annotation from ProductId to enable builder pattern usage during compilation (`#186 <https://github.com/qbicsoftware/data-model-lib/issues/186>`_)
```

Comment:
Did we not want the PR here as well?

Reply:
I think we prefer issue over PR but we dont have an issue for all PRs